crates/biome_js_parser/src/syntax/typescript/types.rs

```diff
@@ -1709,6 +1674,58 @@ fn parse_ts_constructor_type(p: &mut JsParser, context: TypeContext) -> ParsedSy
     Present(m.complete(p, TS_CONSTRUCTOR_TYPE))
 }
 
+// test ts ts_import_type_arguments
+// type A = import("foo");
+// type B = import("foo", "bar");
+// type C = typeof import("foo");
```

Comment:
We moved away from this testing. Now just create a file inside the specs and run `cargo t` to generate the snapshot

Reply:
lol, I asked why I panic here when I execute `cargo codegen test` recently. i will update later~ now the parser js test case is align with css parser.

Reply:
Exactly. The comments in the source code are still there because the PR was too big. We will have to delete them eventually